socorro/cron/jobs/matviews.py

```diff
@@ -68,16 +68,6 @@ class SignaturesCronApp(_MatViewBackfillBase):
     depends_on = ('reports-clean',)
 
 
-class TCBSCronApp(_MatViewBackfillBase):
-    proc_name = 'update_tcbs'
```

Comment:
If we're removing all these cron jobs, can we remove the related postgres procedures, too?